src/workerd/api/tests/compat-flags-test.js

```diff
@@ -0,0 +1,38 @@
+import {
+  ok,
+  strictEqual,
+  throws,
+} from 'node:assert';
+
+import { compatibilityFlags } from 'cloudflare:workers';
+
+const kOn = 'on';
+const kOff = 'off';
+
+export const compatFlagsTest = {
+  test() {
+    throws(() => compatibilityFlags.no_nodejs_compat_v2 = "...");
+    throws(() => compatibilityFlags.not_a_real_compat_flag = "...");
+//    throws(() => { delete compatibilityFlags['nodejs_compat_v2']; });
```

Comment:
```suggestion
```

Reply:
Actually, this should go the other direction... the line should not be commented out.